filters/datetime.js

```diff
@@ -0,0 +1,8 @@
+import Vue from 'vue'
+import moment from 'moment'
+
+Vue.filter('datetime', value => value ? moment(String(value)).local().format('YYYY-MM-DD HH:mm') : '')
+
+Vue.filter('date', value => value ? moment(String(value)).format('YYYY-MM-DD') : '')
+
+Vue.filter('time', value => value ? moment(String(value)).format('HH:mm') : '')
```

Comment:
did you read the card https://trello.com/c/Ak5saPN0/23-create-a-global-datetime-filter carefully?

I asked this file to export the filter functions not to register them globally, registration into Vue should be done in plugins/global-filters.js. it is done this way because we will add more filters into ~/filters/ folder but they will not necerray be registered globally :
> ## Filters:
> Created  `~/filters/datetime.js` files that export all date / time related filters. export: `datetime`, `date` &  `time` 

Do not wrap the `value` with string as it can be a `Date` object. No need to call `.local()`. be default moment render to local

```suggestion
import moment from 'moment'

function datetime(value) {
  return value ? moment(value).format('YYYY-MM-DD HH:mm') : '' 
}

function date(value) {
  return value ? moment(value).format('YYYY-MM-DD') : ''
}

function time(value) {
  return value ? moment(value).format('HH:mm') : ''
}

export { datetime, date, time } 
```